src/prototypes/speaking-event/example/multiple-speakers.twig

```diff
@@ -27,10 +27,27 @@
           <p>
             If you have a website—particularly one that generates revenue for your organization—you need a Progressive Web App. So where do you begin? How do you decide which features of a Progressive Web App make sense for your users? What tools can make the process easier (or harder)? In this practical session, Jason will guide you through the key design decisions you’ll need to make about your Progressive Web App and how those decisions impact the scope of your project. He'll also teach you how to avoid common pitfalls and help you take full advantage of Progressive Web App technology.
           </p>
+          <div class="u-pad-block-start-4">
+            {% include '@cloudfour/components/heading/heading.twig' with {
+              level: 1,
+              tag_name: 'h2',
+              content: 'Resources',
+            } only %}
+          </div>
           {% include '@cloudfour/objects/embed/demo/image.twig' with {
             "class": "o-embed--wide",
             "src": deck
           } only %}
+          <div class="button-columns">
+            {% include '@cloudfour/components/button/button.twig' with {
+              "content_start_icon": "brands/github",
+              "label": "GitHub"
+            } only %}
+            {% include '@cloudfour/components/button/button.twig' with {
+              "content_start_icon": "link",
+              "label": "Progressive Web Apps"
+            } only %}
+          </div>
```

Comment:
Could you use a Button Group? https://v-next--cloudfour-patterns.netlify.app/?path=/docs/objects-button-group--grow

Reply:
Here's what it looks like using the Button Group pattern:

![Screen Shot 2022-03-31 at 3 58 58 PM](https://user-images.githubusercontent.com/42841342/161163288-9d735304-393b-4188-bba0-d07df12d0361.png)

Reply:
Hmm, I'm not sure if that's intentional or not! Maybe we should make some alterations or a new option or modifier for that pattern?

Reply:
I changed the flex property so that the buttons are the same size on large screens. On small screens they retain their default size. It looks like this:

![Screen Shot 2022-03-31 at 4 14 40 PM](https://user-images.githubusercontent.com/42841342/161164673-bb746538-31b0-47a7-a271-22f4e6909a2d.png)
![Screen Shot 2022-03-31 at 4 14 58 PM](https://user-images.githubusercontent.com/42841342/161164676-ff3167ab-2f5b-4fd2-a03d-44c82fd11f79.png)

Reply:
That looks nice!

Reply:
@dromo77 Maybe! Can you coordinate with @gerardo-rodriguez to double check? This looks like his Object that he made originally, and he can tell us if the behavior you observed was intentional. He may also have an opinion on whether or not this should be baked in or a modifier since he'll know better why it was created in the first place.

Reply:
Thanks @tylersticka! @gerardo-rodriguez I'm trying to use the Button Group object to display the supporting materials on the "Talks" page. However, with the current `o-button-group--grow` modifier the buttons are different sizes. I added some custom CSS to the prototype to make the buttons the same size. I'd love your opinion, should I add a new modifier to support this behavior?

Reply:
@dromo77 I think you found a bug with the Button Group pattern! The intent was to have them be the same size. 🙂

Reply:
Thanks @gerardo-rodriguez! Should that be the default behavior even without the `--grow` modifier?

Reply:
@dromo77 Mmm...good question. I suppose it doesn't matter as long as we have the ability to have both options? Either way, we need a modifier. So I don't have a preference. We have only _barely_ started using this pattern so if it seems we always add the grow modifier, then maybe that should be the default! 🙂